src/components/LastDonation.jsx

```diff
@@ -6,6 +6,6 @@ export const LastDonation = (props) => {
   const headline = props.params.get('headline') || 'Letzte Spende'
   return <div className='truncate-with-ellipsis'>
     <HeadlineWithBr content={headline} />
-    {formatCents(props.data.donated_amount_in_cents)} von <Author {...props.data.author} />
+    {formatCents(props.data.donated_amount_in_cents)} <br/> <Author {...props.data.author} />
```

Comment:
```suggestion
    <Author {...props.data.author} /><br/>{formatCents(props.data.donated_amount_in_cents)}
```

Der Plan war eher, zu sagen:
>Danke:
>NAME
>AMOUNT